sct-app/src/main/java/org/lfenergy/compas/sct/app/SclAutomationService.java

```diff
@@ -57,7 +57,7 @@ public SCL createSCD(@NonNull SCL ssd, @NonNull HeaderDTO headerDTO, List<SCL> s
             sclEditor.addHistoryItem(scd, hItem.getWho(), hItem.getWhat(), hItem.getWhy());
         }
         substationEditor.addSubstation(scd, ssd);
-        sclEditor.importSTDElementsInSCD(scd, stds, SUB_NETWORK_TYPES);
```

Comment:
You should delete the constant  `public static final List<SubNetworkTypeDTO> SUB_NETWORK_TYPES` since its no longer used.

Reply:
done